tests/spec/core/dfn-spec.js

```diff
@@ -30,17 +30,28 @@ describe("Core — Definitions", () => {
           <a>partial inner code</a>
         </section>`,
     };
+
+    const getNodeArray = NodeList => Array.prototype.slice.call(NodeList);
```

Comment:
1. camelCase is preferred for variable names, so `NodeList` -> `nodeList`
2. `[...nodeList]` is preferred over `Array.prototype.slice.call(nodeList)` as it's much shorter. We don't even need a helper function then.